src/ui/qgisapp.ui

```diff
@@ -609,6 +638,17 @@
    <addaction name="mActionNewGeoPackageLayer"/>
    <addaction name="mActionNewMemoryLayer"/>
   </widget>
+  <widget class="QToolBar" name="mRegularShapeDigitizeToolBar">
+   <property name="windowTitle">
+    <string>toolBar</string>
```

Comment:
@lbartoletti the name of the toolbar is not specified.
Also, I'm not sure it makes sense to add a new tool bar, I would rather integrate it in the digitizing toolbar (since you can customize it), but not 100% convinced.